test/_unit/behaviours/get-section.spec.js

```diff
@@ -0,0 +1,314 @@
+'use strict';
+
+const Controller = require('hof').controller;
+const Behaviour = require('../../../apps/paf/behaviours/get-section');
+
+describe('apps/paf/behaviours/get-section', () => {
+  describe('configure', () => {
+    let controller;
+    let req;
+    let res;
+    let sandbox;
+
+    beforeEach(done => {
+
+      req = reqres.req();
+      res = reqres.res();
+      req.sessionModel.attributes.steps = [];
+
+      const GetSectionController = Behaviour(Controller);
+      controller = new GetSectionController({ template: 'index', route: '/index' });
+      controller._configure(req, res, done);
+    });
+
+
+    describe(`sets the 'next' property for pages preceeding the first page of each section`, () => {
```

Comment:
This is good just maybe a bit wordy.  I was taught to write test names in simple narrative form and less technical so that anyone can understand especially junior developers. Maybe rewrite the describe as
describe(`sets the next page according to the section`)

Reply:
I have amended this as suggested now, thanks